### PR TITLE
Create fuel mapping table

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

**Comment by wenzowski** [comment]
rebased

File: deploy/table_fuel_mapping.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+begin;
+
+create table ggircs_swrs.fuel_mapping (
+  id integer generated always as identity primary key,
+  fuel_type varchar(1000),
+  n_fuel_type varchar(1000),
+  state varchar(1000),
+  carbon_taxed varchar(1000),
+  cta_mapping varchar(1000),
+  cta_rate_30_tCO2e numeric,
```
[the remark this answers is not in the document]
Comment (wenzowski) [reply deploy/table_fuel_mapping.sql]: if we merge this as is...can we at least get a bug opened to track the tech debt here?

[the remark this answers is not in the document]
Comment (wenzowski) [reply deploy/table_fuel_mapping.sql]: @hamzajaved can you explain how these rates apply or do we need to loop in Andrea?

[the remark this answers is not in the document]
Comment (dleard) [reply deploy/table_fuel_mapping.sql]: @wenzowski @hamzajaved 
Michael @Wenzows found this plan for the carbon tax rates: https://www2.gov.bc.ca/gov/content/environment/climate-change/planning-and-action/carbon-tax

It looks like the mapping should be done on reporting year as the rates increase by $5/tCO2e per year to a cap of $50/tCO2e by 2021.

Mapping:
$30/t - 2017
$35/t - 2018
$40/t - 2019
$45/t - 2020
$50/t - 2021

**Comment by dleard** [comment]
The fuel mapping / implied emission factor tables do not have an exhaustive list of how fuel_types are reported.
select distinct(fuel_type) from ggircs.fuel != fuel types from Fuel_Mapping GHGR to Carbon Tax Act excel file in teams/data.

https://youtrack.button.is/agiles/101-1/102-184?p=101-1%2F102-15&issue=GGIRCS-482

**Pull request merged**